bindings/pyroot_experimental/PyROOT/CMakeLists.txt

```diff
@@ -34,5 +34,8 @@ foreach(py_source ${py_sources})
 endforeach()
 
 ROOT_LINKER_LIBRARY(ROOTPython ${sources} LIBRARIES Core Tree cppyy)
+if(MSVC)
+  set_target_properties(ROOTPython PROPERTIES SUFFIX ".pyd")
```

Comment:
Just for my information: what does this configuration do?

Reply:
Just rename `libROOTPython.dll` into `libROOTPython.pyd` (`pyd` for `Python DLL`). File extensions are important on Windows!

Reply:
Then, with this change and the equivalent ones for cppyy libraries, in Windows we will generate `libROOTPython.pyd`, `libcppyy.pyd` and `libcppyy_backend.dll`? The reason for the difference between the latter and the first two is that the first two are extension modules?

Reply:
Correct. the `.pyd` are for Python, and the `.dll` are for C++ (or `dlopen()` equivalent on Windows)